djangocms_snippet/__init__.py

```diff
@@ -1,3 +1,4 @@
-__version__ = '1.8.3rc1'
+# -*- coding: utf-8 -*-
+__version__ = '1.9.0rc1'
```

Comment:
Please avoid bumping the version of a package as par of other changes.
Bumping the version should happen only on release/x.x.x branches and only contain release related changes like translation, changelog, etc..